Gulpfile.js

```diff
@@ -0,0 +1,267 @@
+/////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+//Useful tasks:
+//
+//      $ gulp server-tests                        - run mocha tests
+//      $ gulp client-tests                        - run qunit test page
+//      $ gulp playground                          - run playground page
+//
+/////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+var babel      = require('babel');
+var gulpBabel  = require('gulp-babel');
+var del        = require('del');
+var eslint     = require('gulp-eslint');
+var fs         = require('fs');
+var gulp       = require('gulp');
+var mocha      = require('gulp-mocha');
+var mustache   = require('gulp-mustache');
+var rename     = require('gulp-rename');
+var sourcemaps = require('gulp-sourcemaps');
+var webmake    = require('gulp-webmake');
+var chalk      = require('chalk');
+var Promise    = require('promise');
+
+function hang () {
+    return new Promise(function () {
+        // NOTE: hang forever
+    });
+}
+
+gulp.task('clean', function () {
```

Comment:
The 'clean' task is not used nowhere now. It seems we should use it in build tasks